docs/sql_service.md

```diff
@@ -0,0 +1,116 @@
+# SQL Service
+
+The SQL Service, developed by Semiotic Labs in collaboration with TheGuild
+and Edge & Node, provides a secure SQL interface for querying a subgraph's entities
+via GraphQL. The `sql` field on the `Query` type exposes inputs so Graph Node can
+receive raw SQL queries (see `sql_field` in `graph/src/schema/api.rs`).
+
+## Configuration
+
+The SQL Service can be enabled or disabled using the `GRAPH_GRAPHQL_ENABLE_SQL_SERVICE`
+environment variable.
+
+- **Environment Variable:** `GRAPH_GRAPHQL_ENABLE_SQL_SERVICE`
+- **Default State:** Off (Disabled)
+- **Purpose:** Enables queries on the `sql()` field of the root query.
+- **Impact on Schema:** Adds a global `SqlInput` type to the GraphQL schema. The `sql`
+field accepts values of this type.
+
+To enable the SQL Service, set the `GRAPH_GRAPHQL_ENABLE_SQL_SERVICE` environment
+variable to `true` or `1`. This allows clients to execute SQL queries using the
+`sql()` field in GraphQL queries.
+
+```bash
+export GRAPH_GRAPHQL_ENABLE_SQL_SERVICE=true
+```
+
+Alternatively, configure the environment variable in your deployment scripts or
+environment setup as needed.
+
+### SQL Coverage
+
+The SQL Service covers a wide range of SQL functionality, allowing you to execute `SELECT`
+queries against your database. It supports basic querying, parameter binding, and result
+formatting into JSON or CSV. 
```

Comment:
Is the result formatting stuff something we've built into graph-node?

Reply:
Yes, I believe output formatting in either CSV or JSON is a sub-feature of the Subgraph:SQL Service.